src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
```

Comment:
No needed to add separate class for only one method.
The logic could be put to Lottery class like this:

`public class Lottery  {
    private static final int COLORS_NUMBER = Colors.values().length ;
    private static final int BALL_NUMBER = Colors.values().length ;
    
    public Ball getRandomBall() {
        Ball ball = new Ball();
        ball.setBallColor(getRandomColor());
        ball.setBallNumber(new Random().nextInt(BALL_NUMBER));
        return ball;
    }
    public String getRandomColor() {
        int index = new Random().nextInt(COLORS_NUMBER);
        return Colors.values()[index].toString();
    }
}`